lab_5_scraper/scraper.py

```diff
@@ -19,6 +72,17 @@ def __init__(self, path_to_config: pathlib.Path) -> None:
         Args:
             path_to_config (pathlib.Path): Path to configuration.
         """
+        self.path_to_config = path_to_config
+        config_content = self._extract_config_content()
+        self._seed_urls = config_content.seed_urls
+        self._headers = config_content.headers
+        self._num_articles = config_content.total_articles
+        self._timeout = config_content.timeout
+        self._encoding = config_content.encoding
+        self.headless_mode = config_content.headless_mode
+        self._should_verify_certificate = config_content.should_verify_certificate
+        self._validate_config_content()
```

Comment:
so you assigned for 5+ variables to cofig content and after this you want to validate this? May be you should change the logic?

Reply:
?